torch/linalg/__init__.py

```diff
@@ -781,7 +781,7 @@
 Computes the first `n` columns of a product of Householder matrices.
 
 Let :math:`\mathbb{K}` be :math:`\mathbb{R}` or :math:`\mathbb{C}`, and
-let :math:`V \in \mathbb{K}^{m \times n}` be a matrix with columns :math:`v_i \in \mathbb{K}^m`
+let :math:`A \in \mathbb{K}^{m \times n}` be a matrix with columns :math:`v_i \in \mathbb{K}^m`
 for :math:`i=1,\ldots,m` with :math:`m \geq n`. Denote by :math:`w_i` the vector resulting from
 zeroing out the first :math:`i-1` components of :math:`v_i` and setting to `1` the :math:`i`-th.
```

Comment:
```suggestion
zeroing out the first :math:`i-1` components of :math:`a_i` and setting to `1` the :math:`i`-th.
```

Reply:
-then it seems reasonable also to rename $w_i \rightarrow b_i$ and who knows where we finish...

__v__ is a good symbol for vectors and renaming the function argument looks like the Right Thing to do.
But I am not sure about the collateral damage.

Reply:
If we are renaming the input matrix `A`, we should do the same for the columns.

Reply:
The first positional argument in the source is in fact "input", so renaming A->V in the docs should probably not harm anybody?

Reply:
A is a better name as it's the one we use all throughout the `torch.linalg` docs. `V` we just use it in `linalg.eig` to represent the eigenvectors.

Reply:
> A is a better name as it's the one we use all throughout the `torch.linalg` docs for the input matrix. `V` we just use it in `linalg.eig` to represent the eigenvectors.

Updated as discussed. Please see if it is OK now.
And sorry for the delay, was busy with something else